src/main/java/com/cocodan/triplan/schedule/controller/ScheduleController.java

```diff
@@ -64,7 +78,7 @@
     }
 
     // 투표
-    @PostMapping("/{scheduleId}/voting")
+    @PostMapping("/{scheduleId}/votings/")
```

Comment:
```suggestion
    @PostMapping("/{scheduleId}/votings")
```
이런 오타는 알아서 잡아준다고 기억하고 있습니다만, 그래도 고쳐주면 좀더 보기좋을거같습니다

Reply:
👍